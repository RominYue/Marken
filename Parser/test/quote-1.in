>   This is a
> quoted paragraph.