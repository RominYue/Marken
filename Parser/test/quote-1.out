<blockquote><p>This is a
quoted paragraph.</p></blockquote>